### ISSUE TITLE
argument "alpha" no effect 

### ISSUE THREAD

**Issue by leeyupeng119** [jheinen/GR.jl#262]
My Julia version is 1.2.0 
GR package version is 0.41.0

> using GR
n = 300
x = rand(n)
y = rand(n)
area = pi .* (15 .* rand(n)).^2
scatter(x, y, area, 1:n, alpha=0.1)
savefig("output.png")

my output.png is here
![image](https://user-images.githubusercontent.com/55687784/67357484-f3beb180-f58f-11e9-9759-a58638094762.png)

the argument _alpha_ is no effect.

**Comment by heliosdrm** [comment]
The argument does have the desired effect when you call the function `scatter`, but it seems that the `alpha` attribute is not kept in the global variables that are used in the workflow of `savefig` later on; that's the source of the problem.

However, I find it difficult to track where this happens exactly and how it could be fixed. I'd say it has to do with how the globals `plt.kvs`, `plt.obj` and `ctx` are copied to each other and modified during the creation and drawing of plots. Unfortunately that process is rather involved and I cannot tell much more than that. Maybe it's something in `jlgr.create_context` or `jlgr.restore_context`.

**Comment by jheinen** [comment]
Yes, this stuff (context, figures, ...) needs to be completely reimplemented. I hope I can finally find the time ...

**Comment by leeyupeng119** [comment]
> Yes, this stuff (context, figures, ...) needs to be completely reimplemented. I hope I can finally find the time ...

Ok, Thank you for the reply!

**Comment by heliosdrm** [comment]
Maybe this was fixed by de79425 (released in GR 0.46.0)?
